proposals/flexible-vectors/BinaryFlexibleVectors.md

```diff
@@ -0,0 +1,313 @@
+Binary encoding of Flexible Vector operations
+=============================================
+
+Vector value types:
+
+```
+valtype ::= ...
+          | 0x7A => vec.i8
+          | 0x79 => vec.i16
+          | 0x78 => vec.i32
+          | 0x77 => vec.i64
+          | 0x76 => vec.f32
+          | 0x75 => vec.f64
+```
+
+## Instruction encoding
+
+Instructions are prefixed with a byte determining which lane type the
+instruction supports, the actual opcode is encoded by variable-length unsigned
+integer:
+
+
+```
+instr ::= ...
+        | OxFA vecop:varuint32 ... => vec.i8
```

Comment:
typo, 0x7A

Reply:
also in the instructions below

Reply:
Ouch, not sure how I missed that, it got copy-pasted all over the place 😞

Reply:
Another comical bit about this typo is that I managed to type letter O instead of a zero. Fixed in the update.